lms/static/sass/base/_variables.scss

```diff
@@ -463,7 +463,7 @@ $border-color-l4: $m-gray-l4;
 // MISC: links and buttons
 $link-color: $blue;
 $link-color-d1: $blue;
-$link-hover: $pink;
+$link-hover: $blue-l1; // Was formerly $pink, but is synced up with our Pattern Library http://ux.edx.org/elements/colors/
```

Comment:
No need to say that this was formerly ``$pink`` in your comment. Just mirror your other uxpl comments.